web/src/components/datasets/DatasetDetailPage.tsx

```diff
@@ -116,9 +118,32 @@ const DatasetDetailPage: FunctionComponent<IProps> = (props) => {
   }
 
   const firstVersion = versions[0]
-  const { name, description } = firstVersion
+  const { name, tags, description } = firstVersion
   const facetsStatus = datasetFacetsStatus(firstVersion.facets)
 
+  const formatTags = (tags: string[], tag_desc: Tag[]) => {
```

Comment:
This isn't strictly needed as its been superseded by DatasetTags.

Reply:
I made this code a bit cleaner.